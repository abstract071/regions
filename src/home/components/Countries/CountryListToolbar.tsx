import React from 'react'
import { useHistory } from 'react-router-dom'
import Toolbar from '@material-ui/core/Toolbar'
import Typography from '@material-ui/core/Typography'
import Button from '@material-ui/core/Button'


const CountryListToolbar: React.FC = () => {
  const history = useHistory()

  return (
    <Toolbar className="toolbar">
      <Typography variant="h4" id="tableTitle">
        Region's countries
      </Typography>
      <Button
        variant="outlined"
        color="primary"
        onClick={ () => history.goBack() }
      >
        Back
      </Button>
    </Toolbar>
  )
}

export default CountryListToolbar
