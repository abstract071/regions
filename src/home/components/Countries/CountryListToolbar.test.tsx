import React from 'react'
import { shallow } from 'enzyme'

import CountryListToolbar from './CountryListToolbar'


describe( 'CountryListToolbar', () => {
  let shallowedComponent: any = null

  beforeEach( () => {
    shallowedComponent = shallow(
      <CountryListToolbar />
    )
  } )

  it( 'should match snapshot', () => {
    expect( shallowedComponent ).toMatchSnapshot()
  } )
} )
