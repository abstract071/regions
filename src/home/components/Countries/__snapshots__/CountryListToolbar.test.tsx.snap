// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountryListToolbar should match snapshot 1`] = `ShallowWrapper {}`;
